roles/edpm_nova/molecule/default/verify.yml

```diff
@@ -40,3 +40,17 @@
         - "Copying /var/lib/kolla/config_files/nova-blank.conf to /etc/nova/nova.conf"
         - "Copying /var/lib/kolla/config_files/01-nova.conf to /etc/nova/nova.conf.d/01-nova.conf"
         - "Copying /var/lib/kolla/config_files/02-nova-log.conf to /etc/nova/nova.conf.d/02-nova-log.conf"
+
+    - name: Lookup the nova user
+      ansible.builtin.shell:
```

Comment:
in ansible code the best is to avoid shell scripts when possible. I'd rather use here the builtin getent modul, something like this:

    - name: Check if user exists
      ansible.builtin.getent:
        database: passwd
        key: nova

This fails if nova user doesn't exist, but 'register: nova_user' can be used as well.